.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: Build CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  gqltools:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.18
+
+      - name: Get Go dependencies
```

Comment:
I don't think this step is necessary.  The `go test` command should download dependencies automatically and I don't have it in either `csproto` or `perseus`.